io.elementary.BaseApp.yml

```diff
@@ -0,0 +1,72 @@
+id:                   io.elementary.BaseApp
```

Comment:
Don't do this weird indenting, nothing else does it.

Reply:
I’m actually planning to switch my apps to this indenting, it looks so much cleaner IMO.

Reply:
Just used the default atom-beautify plugin, I can revert that but it looks really clean

Reply:
Ah yeah, didn’t think about the diff.

Reply:
I created an entry just for this conversation: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

(I assume other maintainers will agree)

Reply:
@TingPing Fixed.